### ISSUE TITLE
Return null for empty fields

### ISSUE THREAD

**Issue by OmgImAlexis** [sebhildebrandt/systeminformation#329]
**Is your feature request related to a problem? Please describe.**
I'm using this library with graphql and finding myself having todo a lot of type conversion which just seems unnecessary, one of the main ones I've come across is non-resolvable fields returning an empty string.

```js
const si = require('systeminformation');
si.cpu().then(console.log);
{
  manufacturer: 'Intel®',
  brand: 'Core™ i5 M 520',
  vendor: 'GenuineIntel',
  family: '6',
  model: '37',
  stepping: '5',
  revision: '',
  voltage: '',
  speed: '2.40',
  speedmin: '2.40',
  speedmax: '2.40',
  governor: '',
  cores: 4,
  physicalCores: 2,
  processors: 1,
  socket: '',
  cache: { l1d: 32768, l1i: 32768, l2: 262144, l3: 3145728 }
}
```

**Describe the solution you'd like**
Fields that can't be resolved to return null not an empty string.

```js
{
  manufacturer: 'Intel®',
  brand: 'Core™ i5 M 520',
  vendor: 'GenuineIntel',
  family: '6',
  model: '37',
  stepping: '5',
  revision: null,
  voltage: null,
  speed: '2.40',
  speedmin: '2.40',
  speedmax: '2.40',
  governor: null,
  cores: 4,
  physicalCores: 2,
  processors: 1,
  socket: null,
  cache: { l1d: 32768, l1i: 32768, l2: 262144, l3: 3145728 }
}
```

**Comment by sebhildebrandt** [comment]
@OmgImAlexis I thought about this and yes, this should be the right way to return back empty values. As I would consider this as a breaking change, I will add this to the next major version 5.x

**Comment by sebhildebrandt** [comment]
@OmgImAlexis I just released a new version 5.0 (with soma new functions and also some breaking changes). Here this should be done now mostly ... if you discover more spots where this is missing, please feel free to contact me.
